rust/operator-binary/src/druid_controller.rs

```diff
@@ -734,6 +735,8 @@ fn build_rolegroup_config_map(
                 };
 
                 if let Some(s3) = s3_conn {
+                    conf.insert(S3_REGION_NAME.to_string(), Some(s3.region.name.to_string()));
```

Comment:
I think according to https://druid.apache.org/docs/latest/development/extensions-core/s3/#aws-region `aws.region` needs to go into the JVM args, not the Druid config

Reply:
It does, but that also seems strange to be since you might want to connect to buckets in multiple regions.

I think this needs some testing.